pycodestyle.py

```diff
@@ -2105,7 +2105,16 @@ def read_config(options, args, arglist, parser):
 
     parent = tail = args and os.path.abspath(os.path.commonprefix(args))
     while tail:
-        if config.read(os.path.join(parent, fn) for fn in PROJECT_CONFIG):
+        try:
+            files_read = config.read(os.path.join(parent, fn)
+                                     for fn in PROJECT_CONFIG)
+        except configparser.ParsingError:
+            if options.verbose:
+                print('failed to read local configuration files due to a '
```

Comment:
@sigmavirus24 -- It would be nice if we also printed out the list of files: `os.path.join(parent, fn) for fn in PROJECT_CONFIG` so that a user could narrow down which ones caused the ParsingError.